web/js/map-OpenLayers.js

```diff
@@ -845,7 +845,19 @@ $.extend(fixmystreet.utils, {
                 // This option is thankfully used by them both
                 numZoomLevels: fixmystreet.numZoomLevels
             }, fixmystreet.layer_options[i]);
-            if (fixmystreet.layer_options[i].matrixIds) {
+            if (fixmystreet.layer_options[i].wms_version) {
```

Comment:
Worth sticking fixmystreet.layer_options[i] in another var if it's going to be used a lot more?